web/js-file.js

```diff
@@ -1,6 +1,8 @@
 function funk(a,b) {
```

Comment:
Reporter: JSHINT
Rule: ?
Severity: WARN
File: ../../../web/js-file.js L9
 

&apos;c&apos; is not defined.: return c;

<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>